src/stores/plugins/api.ts

```diff
@@ -92,7 +92,7 @@ export default function api():IPluginApi {
   /**
    * Handles axios error responses.
    */
-  function handleError(error: AxiosError): Promise<AxiosError> {
+  function handleError(error: AxiosError<any>): Promise<AxiosError> { // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
question: would it work to use `unknown` instead of `any` here? `unknown` would be the better term in my opinion. @see https://mariusschulz.com/blog/the-unknown-type-in-typescript